.changeset/twelve-bobcats-sort.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/design-tokens': patch
+'@twilio-paste/core': patch
```

Comment:
suggestion: I think technically this should be a major if we're moving things. Might be pretty important to call out for Flex too _just in case_